### DISCUSSION TITLE
Subband CQI Query

### DISCUSSION THREAD

**Discussion by aibtw** [srsran/srsRAN_Project#320] [General Help] [answered]
Hi, 
I have tried both the 4G and the 5G project, and noticed that, unlike the 4G codebase, the 5G codebase **doesn't mention subband cqi**. 

I am just wondering, is this related to the standard, or is it just not implemented and will be implemented in the future?

**Comment by arman-maghsoudnia** [comment]
Hey @aibtw, this link might be the answer to your question: https://github.com/srsran/srsRAN_Project/discussions/329

**Comment by herlesupreeth** [comment, marked as answer]
> I am just wondering, is this related to the standard, or is it just not implemented and will be implemented in the future?

It will be implemented in future. Dont know the timeline yet
